RELEASE.md

```diff
@@ -0,0 +1,52 @@
+# How to make a release
+
+`chartpress` is a package [available on
+PyPI](https://pypi.org/project/chartpress/). These are instructions on how to
+make a release on PyPI.
+
+For you to follow along according to these instructions, you need:
+- To be a maintainer of the [PyPI chartpress
+  project](https://pypi.org/project/chartpress/).
+- To have push rights to the chartpress GitHub repository.
```

Comment:
In case someone is reading this README in a fork:
```suggestion
- To have push rights to the [chartpress GitHub repository](https://github.com/jupyterhub/chartpress).
```